src/systems/systems/src/pressurization/pressure_valve.rs

```diff
@@ -1,44 +1,99 @@
-use crate::simulation::UpdateContext;
+use crate::{shared::ControllerSignal, simulation::UpdateContext};
 
-use super::PressureValveActuator;
+use super::{CabinPressure, OutflowValveActuator, PressurizationOverheadPanel};
 
 use std::time::Duration;
 use uom::si::{f64::*, ratio::percent};
 
+pub(super) enum PressureValveSignal {
+    Open,
+    Close,
+    Neutral,
+}
+
 pub(super) struct PressureValve {
     open_amount: Ratio,
     target_open: Ratio,
     full_travel_time: Duration,
+    manual_travel_time: Duration,
 }
 
 impl PressureValve {
-    pub fn new() -> Self {
+    pub fn new_open() -> Self {
         Self {
             open_amount: Ratio::new::<percent>(100.),
             target_open: Ratio::new::<percent>(100.),
             full_travel_time: Duration::from_secs(4),
+            manual_travel_time: Duration::from_secs(60),
         }
     }
 
-    pub fn update<T: PressureValveActuator>(&mut self, context: &UpdateContext, actuator: &T) {
-        self.target_open = actuator.target_valve_position();
-        if self.open_amount < self.target_open {
-            self.open_amount += Ratio::new::<percent>(
-                self.get_valve_change_for_delta(context)
-                    .min(self.target_open.get::<percent>() - self.open_amount.get::<percent>()),
-            );
-        } else if self.open_amount > self.target_open {
-            self.open_amount -= Ratio::new::<percent>(
-                self.get_valve_change_for_delta(context)
-                    .min(self.open_amount.get::<percent>() - self.target_open.get::<percent>()),
-            );
+    pub fn new_closed() -> Self {
+        Self {
+            open_amount: Ratio::new::<percent>(0.),
+            target_open: Ratio::new::<percent>(0.),
+            full_travel_time: Duration::from_secs(1),
+            manual_travel_time: Duration::from_secs(1),
+        }
+    }
```

Comment:
I find it surprising that these constructors default to different `manual_travel_time` (60 seconds vs. 1 second).
Do you need both? Maybe this should be supplied as a parameter?

Also consider the `Default` trait if there is a "common" way of initializing the pressure valves (e.g. usually closed when starting).

Reply:
The logic here was that the `new_open()` was applicable to the outflow valve, and the `new_closed()` was applicable to the safety valve. These both being pressure valves but with different physical mechanisms have different travel times. 

I've changed the names of the constructors to be clearer now. If in the future for any reason at all we decide to use this `PressureValve` type anywhere else we can rethink the constructor naming scheme.